clients/lsp-galaxy.el

```diff
@@ -0,0 +1,86 @@
+;;; lsp-galaxy.el --- description -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 emacs-lsp maintainers
+
+;; Author: emacs-lsp maintainers
+;; Keywords: lsp, php
```

Comment:
nit: fix keywords

Reply:
should I change the authors here too, or are all contributors maintainers? (I'm also happy to leave it as it is, I just feel I should not burden you with the task of maintaining this extension if it falls out of date)

Reply:
You can leave it as it is or put your name.